chassis.yaml

```diff
@@ -0,0 +1,10 @@
+# Extension version.
+#
+# All modern extensions should use 2 here. Version 1 uses a chassis.pp
+# instead, which requires a deprecated feature of Puppet.
+#
+# Values: 2
+version: 2
+
+dependencies:
+  - query-monitor
```

Comment:
This should be `querymonitor`, i.e. no `-`

https://github.com/Chassis/Query-Monitor/blob/master/chassis.pp#L1